src/vehicleUrban.cpp

```diff
@@ -30,10 +29,9 @@ CanSensorAccessories canSensorAccessories(canInterface, CAN_ACC_STATUS);
 CanSensorSteering steering(canInterface);
 
 // Bms
-CanSensorTinyBms tinyBms(canInterface, 25);
 CanSensorOrionBms orionBms(canInterface);
-CanSensorBms* bms = DEFAULT_BMS == BmsManager::BmsOption::Orion ? (CanSensorBms*)(&orionBms) : (CanSensorBms*)(&tinyBms);
-BmsManager bmsManager(&bms, &orionBms, &tinyBms, DEFAULT_BMS);
+CanSensorBms* bms = (CanSensorBms*)(&orionBms);
```

Comment:
If you remove BmsManager, then there is no need to use this bms ptr. You could just rename `orionBms` to `bms` (although you'd have to make some other changes, updating `->` method invocations to `.` and all that)